tests/e2e/support/objects/url-navigation/actions.ts

```diff
@@ -29,9 +30,12 @@ export const navigateToDetailsPanelOfResource = async (
 }
 
 export const openResourceViaUrl = async (args: openResourceViaUrlArgs) => {
-  const { page, resource, user, space } = args
+  const { page, resource, user, space, collabora } = args
```

Comment:
Wouldn't it make sense to pass the editor name as variable instead of a boolean? When want to test the same for OnlyOffice, don't we? 🤔

Reply:
I just didn't find way how to open file in the OnlyOffice using url (wanted to ask about it tomorrow.)
Even if I set the `app=OnlyOffice` parameter, the file still opens in `collabora`

`https://ocis.owncloud.test/external/open-with-web?app=OnlyOffice&fileId=fileuuId`

Reply:
I spoke with @JammingBen. Looks like it's a bug

> Wouldn't it make sense to pass the editor name as variable instead of a boolean? When want to test the same for OnlyOffice, don't we?

sure, I'll change it

Reply:
That would be another regression, right? Super important to have the test for both Collabora and OnlyOffice then 😁

Reply:
created bug for that: https://github.com/owncloud/web/issues/10422